src/game/Utils/Font_Control.h

```diff
@@ -45,52 +45,53 @@ extern SGPFont gpHugeFont;
 
 #define HUGEFONT		((GameState::getInstance()->isEditorMode() && isEnglishVersion()) ? gpHugeFont : gp16PointArial)
 
-#define FONT_MCOLOR_BLACK	0
-#define FONT_MCOLOR_WHITE	208
-#define FONT_MCOLOR_DKWHITE	134
-#define FONT_MCOLOR_DKWHITE2	134
-#define FONT_MCOLOR_LTGRAY	134
-#define FONT_MCOLOR_LTGRAY2	134
-#define FONT_MCOLOR_DKGRAY	136
-#define FONT_MCOLOR_LTBLUE	203
-#define FONT_MCOLOR_LTRED	162
-#define FONT_MCOLOR_RED	163
-#define FONT_MCOLOR_DKRED	164
-#define FONT_MCOLOR_LTGREEN	184
-#define FONT_MCOLOR_LTYELLOW	144
+#define FONT_MCOLOR_TRANSPARENT 0x00000000
```

Comment:
Do we need a RGBA macro akin to RGB to make these a wee bit more readable? Makes it easier to change the values too, but unless we eventually externalize them, everyone that could change them will know how.

Reply:
They were actually changed from old color index to hex RGBA value since that commit.
```
#define FONT_MCOLOR_TRANSPARENT 0x00000000
#define FONT_MCOLOR_BLACK	0x000000FF
#define FONT_MCOLOR_WHITE	0xFFFFFFFF
....
#define FONT_MCOLOR_LTGREEN	0x00CE00FF
#define FONT_MCOLOR_LTYELLOW	0xFFD73FFF
```
I find them pretty obvious when put like that (0xRRGGBBAA), but i can replace them with RGBA macro if you think that's better which was also added after that commit.